modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -111,6 +112,28 @@ def diff(rep_data)
         end
       end
 
+      def self.get_suffix(first_name, last_name)
+        suffixes = %w[H.M H.I.H H.R.H Dr Esq Jr Sr III II I AB BA BS BE BFA BTech LLB BSc MA MS MFA LLM
+                      MLA MBA MSc MEng JD MD DO PharmD DMin PhD EdD DPhil DBA LLD EngD]
```

Comment:
Where did these suffixes come from? Would it be simpler to split the name(s) by spaces and return the last element?

e.g.,

```rb
suffix = "Downey Jr".split(' ').last
=> "Jr"
```

This way has the downside of returning double-barrel surnames, e.g., `"Lloyd Webber".split(' ').last => "Webber"`, but we would include the full surname in the check on [line 124](https://github.com/department-of-veterans-affairs/vets-api/blob/506860dee1d6f66e3c825d60ef5fb8381ce3e767/modules/veteran/app/models/veteran/service/representative.rb#L124-L125) and theoretically find the rep (without entering the logic to strip the suffix).

On the upside, we would not have to maintain possible suffixes and their various forms (punctuation in the middle, different abbreviations, etc.).